FogCreek/Glitch-Community
# Pebble-fragrance: add a table view of the analytics graphs

## Links
* https://pebble-fragrance.glitch.me on team pages
* FogCreek/Glitch-Community-Work#

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/4480480/63125193-ad871980-bf72-11e9-9417-dd7c10d38d10.png)

![image](https://user-images.githubusercontent.com/4480480/63125624-bb896a00-bf73-11e9-893a-b8891e5ca591.png)


## Changes:
* On the team analytics section, I added a button that, when clicked, shows the contents of the line graph in a table so that screenreader users have a better way of accessing the content.

## How To Test:
* Go to one of your team pages, press the button to toggle open the section. Update the filters for analytics; the graph should update too

## Feedback I'm looking for:
CSS improvements? Better ways of generating the table?
